### PR TITLE
Add Logging documentation

### PR DESCRIPTION
**Pull request by alfonsserra**

### CODE REVIEW

**Comment by alfonsserra** [comment]
> Great Job. But a first logging example would be nice to have.

Thank you @quimnoto. I think I have not understood your comment. Could you please be more specific?

**Comment by quimnoto** [comment]
I mean that in the documentation it nice explain how to perform the log from one class, but to make the task more complete it would be nice to have one example in a class such as the UserService.java. It is just a comment.
Maybe before to do it we should have a meting to decide if we are going to use a log coding to identify all the messages logged and what decide the information to log to log the same in all the seeds projects.

**Comment by alfonsserra** [comment]
@quimnoto There is an example at: https://github.com/systelab/seed-springboot/blob/59b0c1bdeb661996dccb04c7067bb1b1d1a46732/src/main/java/com/systelab/seed/service/PatientMaintenanceService.java

File: src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </appender>
+
+    <!-- LOG everything at INFO level -->
+    <root level="info">
+        <appender-ref ref="RollingFile" />
+        <appender-ref ref="Console" />
+    </root>
+
+    <!-- LOG "com.baeldung*" at TRACE level -->
+    <logger name="com.baeldung" level="trace" additivity="false">
```
[the remark this answers is not in the document]
Comment (alfonsserra) [reply src/main/resources/logback-spring.xml]: Good catch!!!! Thank you

**Pull request merged**